CMakeLists.txt

```diff
@@ -3,30 +3,44 @@
 # SPDX-License-Identifier: MIT
 
 cmake_minimum_required (VERSION 3.5)
+project(aklite)
 
-# Export compile_commands.json for clang-tidy
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+# OSTree and DockerApp package manager are mandatory for aktualizr-lite
+set(BUILD_OSTREE ON CACHE BOOL "")
+set(BUILD_DOCKERAPP ON CACHE BOOL "")
```

Comment:
DockerApp isn't required. We actually have at least one platform without docker support (riscv).

Reply:
OK, will make BUILD_DOCKERAPP  back as optional.